viết python class cho footer

jasonit8 opened the pull request.

jasonit8 committed:
thêm banner

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

jasonit8 committed:
sửa mũi tên

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

jasonit8 committed:
sửa lại banner animation

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

jasonit8 committed:
sửa banner animation

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

jasonit8 committed:
xong banner và header, sửa trang news cho khớp js

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

jasonit8 committed:
viết python class cho footer

Signed-off-by: Trọng Hiếu <jasonnfi8@gmail.com>

thucnqt commented:
Good job a ! Em đã xem qua cách a viết sử dụng thư viện BeautifulSoup4 để trích xuất nội dung cần thay đổi trong HTML. A cập nhật thêm giúp em nha, change nhưng k ảnh hưởng đến nội dung template cũ và ghi ra 1 file mới. Nếu được dữ liệu truyền vô từ 1 file .txt nào đó chẳng hạn.

jasonit8 committed:
thay đổi file xuất

Theo đề nghị của Thức, anh vẫn cho chương trình đọc index.html gốc nhưng xuất ra index_newfooter.html.

jasonit8 commented:
Theo đề nghị của Thức, anh vẫn cho chương trình đọc index.html gốc nhưng xuất ra index_newfooter.html.

jasonit8 committed:
thêm hàm to_html để tinh gọn lớp

thucnqt commented:
> Theo đề nghị của Thức, anh vẫn cho chương trình đọc index.html gốc nhưng xuất ra index_newfooter.html.

A fix lỗi: Nếu file tạo ra chưa có thì sao.  Update giúp em nhé.

jasonit8 committed:
sửa input thành index.html - file bắt buộc

sửa input thành index.html - file bắt buộc

thucnqt reviewed (commented):
Good Job !
